blaze/compute/python.py

```diff
@@ -666,6 +667,12 @@ def compute_up(t, seq, **kwargs):
         return take(t.n, seq)
 
 
+@dispatch(Sample, Sequence)
+def compute_up(t, seq, **kwargs):
+    nsamp = t.n if t.n is not None else int(t.frac * len(seq))
+    return random.sample(seq, min(nsamp, len(seq)))
```

Comment:
I'm fine with this as-is, since both Pandas' `sample` and `random.sample` both implement random order when `n == len(seq)`, and it provides a way for users to randomize the order of all rows if that's desired.